api.md

```diff
@@ -1829,22 +1809,16 @@ Update the continuous aggregate view immediately.
 REFRESH MATERIALIZED VIEW contagg_view;
 ```
 
->:TIP: Note that max_interval_per_job and refresh_lag parameter settings are used by the materialization job
-when the REFRESH is run. So the materialization (of the continuous aggregate) does not necessarily include
-all the updates to the hypertable.
-
 ---
 
-## DROP VIEW (Continuous Aggregate) :community_function: [](continuous_aggregate-drop_view)
-Continuous aggregate views can be dropped using `DROP VIEW` statement.
+## DROP MATERIALIZED VIEW (Continuous Aggregate) :community_function: [](continuous_aggregate-drop_view)
+Continuous aggregate views can be dropped using the `DROP MATERIALIZED VIEW` statement.
 
-This deletes the hypertable that stores the materialized data for the
-continuous aggregate; it does not affect the data in the underlying hypertable
-from which the continuous aggregate is derived (i.e., the raw data).  The
-`CASCADE` parameter is required for this command.
+This statement deletes the continuous aggregate and all its internal objects. To also delete other dependent objects, add the `CASCADE` option. Dropping a continuous aggregate does not affect the data in the
```

Comment:
Can you give an example of what dependent objects might be?

"To also delete other dependent objects (e.g., <...>), add the ..."

Reply:
Added an example as subphrase.